### PR TITLE
checking description after editing it

### PR DESCRIPTION
**Pull request by parthp2107**
### Describe your changes :

Checking the description after editing it.

Edit pipeline task description will fail due to: #1941

#
### Type of change :

- [x] Improvement

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [x] I have performed a self-review of my own. 
- [x] I have tagged my reviewers below.
- [x] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

### CODE REVIEW

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/pipelineDetails/PipelineDetailsPageTest.java [hunk 1]
```diff
@@ -122,8 +128,15 @@ public void editTaskDescription() throws InterruptedException {
...
-    Events.sendKeys(webDriver, By.xpath(enterDescription), faker.address().toString());
+    Events.sendKeys(webDriver, By.xpath(enterDescription), editDescription);
     Events.click(webDriver, By.cssSelector("[data-testid='save']"));
+//    webDriver.navigate().refresh();
+//    Thread.sleep(1000);
+//    actions
+//        .moveToElement(webDriver.findElement(By.xpath("//div[@data-testid='description']/button")))
+//        .perform();
+//    Events.click(webDriver, By.xpath("//div[@data-testid='description']/button"));
+//    webDriver.findElement(By.xpath("//*[text()[contains(.,'" + editDescription + "')]] "));
```
Comment (shahsank3t) [inline openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/pipelineDetails/PipelineDetailsPageTest.java:139]: If the comments are not required, then lets remove it

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/tableDetails/TableDetailsPageTest.java [hunk 2]
```diff
@@ -103,8 +109,18 @@ public void searchColumnAndEditDescription() throws InterruptedException {
...
+//    webDriver.navigate().refresh();
+//    wait.until(
+//        ExpectedConditions.elementToBeClickable(By.cssSelector("[data-testid='searchbar']")));
+//    Events.sendKeys(webDriver, By.cssSelector("[data-testid='searchbar']"), "address1");
+//    Thread.sleep(2000);
+//    actions
+//        .moveToElement(webDriver.findElement(By.xpath("//div[@data-testid='description']//button")))
+//        .perform();
+//    Events.click(webDriver, By.xpath("//div[@data-testid='description']//button"));
+//    webDriver.findElement(By.xpath("//*[text()[contains(.,'" + editDescription + "')]] "));
```
Comment (shahsank3t) [inline openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/tableDetails/TableDetailsPageTest.java:123]: same here

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/pipelineDetails/PipelineDetailsPageTest.java (hunk 1 above)
Comment (parthp2107) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/pipelineDetails/PipelineDetailsPageTest.java]: Addressed in the latest commit.

File: openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/tableDetails/TableDetailsPageTest.java (hunk 2 above)
Comment (parthp2107) [reply openmetadata-ui/src/test/java/org/openmetadata/catalog/selenium/pages/tableDetails/TableDetailsPageTest.java]: Addressed in the latest commit.

**Review by shahsank3t — APPROVED** [review]

**Pull request merged**